fe/fe-core/src/main/java/com/starrocks/analysis/ShowAlterStmt.java

```diff
@@ -149,9 +156,6 @@ private void analyzeSubPredicate(Expr subExpr) throws AnalysisException {
     public void analyze(Analyzer analyzer) throws AnalysisException, UserException {
         //first analyze 
         analyzeSyntax(analyzer);
```

Comment:
The functionality of analyzeSyntax has not been moved into the new Analyzer. This part of the code will be lost in the new Analyzer